include/rbs/util/rbs_assert.h

```diff
@@ -0,0 +1,8 @@
+#ifndef RBS_ASSERT_H
+#define RBS_ASSERT_H
+
+#include <stddef.h>
+
+void rbs_assert(int condition, const char *fmt, ...);
```

Comment:
```suggestion
void rbs_assert(int condition, const char *fmt, ...) RBS_ATTRIBUTE_FORMAT(2, 3);
```

Context:

https://github.com/Shopify/rbs/blob/5afb8798ba8c789d6d6b11ed36a5aeedf623ed3e/include/rbs/defines.h#L18-L34

Compare:

https://github.com/Shopify/rbs/blob/5afb8798ba8c789d6d6b11ed36a5aeedf623ed3e/include/rbs/parser.h#L6C87-L6C114